src/uopool/memory_mempool.rs

```diff
@@ -14,75 +10,70 @@ use super::Mempool;
 #[derive(Default, Educe)]
 #[educe(Debug)]
 pub struct MemoryMempool {
-    user_operations: Arc<RwLock<HashMap<UserOperationHash, UserOperation>>>, // user_operation_hash -> user_operation
-    user_operations_by_sender: Arc<RwLock<HashMap<Address, HashSet<UserOperationHash>>>>, // sender -> user_operations
+    user_operations: HashMap<UserOperationHash, UserOperation>, // user_operation_hash -> user_operation
+    user_operations_by_sender: HashMap<Address, HashSet<UserOperationHash>>, // sender -> user_operations
```

Comment:
I am curious what is the intention to remove the `RwLock` primitives here?

Reply:
I don't think they are needed and bring some additional overhead. We have another `RwLock` here: https://github.com/Vid201/aa-bundler/blob/main/src/uopool/services/uopool.rs#L25. Mempools are always accessed through that `RwLock`, so the whole mempool should be anyway thread-safe.

What do you think?

Reply:
Oh. That makes sense. If there are some performance problem in the future, we could make lock scope smaller at that time. Cool.

Reply:
Exactly, great!